lib/add-rm-pkg-deps.js

```diff
@@ -55,7 +55,7 @@ const addSingle = ({pkg, spec, saveBundle, saveType, path}) => {
   pkg[type] = pkg[type] || {}
   if (rawSpec !== '' || pkg[type][name] === undefined) {
     // if we're in global mode, file specs are based on cwd, not arb path
```

Comment:
Is this comment still valid?  Should be for all file specs, not just global ones.

Reply:
wow this comment is just a lie! will update it